_episodes/51.md

```diff
@@ -0,0 +1,155 @@
+
+[Trino 425](https://trino.io/docs/current/release/release-425.html)
+
+* Improved performance of `GROUP BY`.
+* Support for check constraints in `MERGE` for Delta Lake connector.
+* Support for the Decimal128 in MongoDB connector.
+
+[Trino 426](https://trino.io/docs/current/release/release-426.html)
+
+* Support for `SET/RESET SESSION AUTHORIZATION`.
+* Improved performance of aggregations over decimal values.
+* Support for `TRUNCATE TABLE` in Delta Lake connector.
+* Support for Databricks 13.3 LTS.
+
+[Trino 427](https://trino.io/docs/current/release/release-427.html)
+
+* Improved performance for `GROUP BY` and `DISTINCT`.
+* Support for pushing down `UPDATE` statements into connectors.
+* Support for reading Delta Lake tables with Deletion Vectors.
+* Faster writing to Parquet files in Delta Lake and Iceberg.
+* Support for querying tags in Iceberg.
+
+## Concept of the episode: PopSQL
+
+It may be familiar to some of our viewers to describe an environment where
+key queries and dashboards are buried in someone's personal workspace, and you
+have to go ask them directly every time you want to check on your metrics.
+When you're running a world-class, highly-performant query engine like Trino and
+investing time and resources into maintaining it, shouldn't you treat your
+queries like a first-class, collaborative, versioned system, too?
+
+[PopSQL](https://popsql.com/), a playful spin on the word popsicle, solves the
+sadness that is disorganized and siloed insights by centralizing queries into a
+platform that has versioning, security, and a suite of collaborative tools
+comparable to Google Drive. Want to work with your teammate on a query? You can
+open up the same editor and see the same thing. Want to see what that query
+someone ran last week was to see how the new feature is doing? It's there. Have
+a suggestion to improve something? Leave a comment. Realize your suggestion was
+wrong and need to undo the change? You can view past versions of the query.
+
+PopSQL and Trino make sense together. PopSQL provides a best-in-class interface
+for organizing, collaborating, and working together on all of your SQL queries
+across the business, and Trino handles running those queries at unparalleled
+speeds. They go hand-in-hand for treating your data and SQL analytics as first
+class citizens. In today's episode, we'll be exploring what PopSQL is, how it
+integrates with Trino, and how the engineers at PopSQL have done some cool
+things with Trino to make the integration better than ever before. We'll start
+with that last one, actually.
+
+## Concept of the episode: A new Node.js adapter for Trino
+
+Trino in the frontend is... a tricky thing. We can go ahead and admit that the
+[Trino web UI](/docs/current/admin/web-interface.html) isn't going to win any
+awards for design or functionality. And while a couple Node-based libraries
+exist out there, including [presto-client-node](https://www.npmjs.com/package/presto-client)
```

Comment:
We initially used this library, but in addition to the lack of streaming support, there's also some missing functionality (e.g. [50x errors throw an uncaught error](https://github.com/tagomoris/presto-client-node/issues/72)) and some idiosyncrasies in the codebase (e.g. [callbacks that have an `err` variable that's never defined](https://github.com/tagomoris/presto-client-node/issues/70)), it's ended up not a great fit.

Reply:
Should we add some of this @colebow ?